lib/LedgerSMB/PGDate.pm

```diff
@@ -252,6 +252,11 @@ sub _formatter {
 # table (being GL>Search-ed without restrictions)
 memoize('_formatter');
 
+# Provide a stringification routine instead of the one inherited
+use overload (
+    q{""}    => 'to_output',
```

Comment:
I think you need `fallback => ` there:

https://perldoc.perl.org/overload#fallback